omnibus/config/software/supermarket-cookbooks.rb

```diff
@@ -41,6 +41,7 @@
     open("#{cookbooks_path}/solo.rb", "w") do |file|
       file.write <<-EOH.gsub(/^ {8}/, '')
         cookbook_path   "#{cookbooks_path}"
+        cache_path "/var/opt/supermarket/local-mode-cache"
```

Comment:
@robbkidd Actually after looking over the `Chef::Config` code it appears just setting `cache_path` is enough as other `*_cache_path` values use this as a base:
https://github.com/chef/chef/blob/master/chef-config/lib/chef-config/config.rb#L263